google_monitoring/uptime_checks.tf

```diff
@@ -7,10 +7,23 @@ resource "google_monitoring_uptime_check_config" "https" {
   user_labels  = each.value.user_labels
 
   http_check {
-    path         = each.value.path
-    port         = 443
-    use_ssl      = true
-    validate_ssl = true
+    path                = each.value.path
+    port                = 443
+    request_method      = each.value.request_method
+    use_ssl             = true
+    validate_ssl        = true
+    content_type        = lookup(each.value, "content_type", null)
+    custom_content_type = lookup(each.value, "custom_content_type", null)
+    body                = lookup(each.value, "body", null)
```

Comment:
Do you know that `null` is a valid value to be passed in here as the default? For example, `content_type` [per the docs](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/monitoring_uptime_check_config#content_type) says possible values are `TYPE_UNSPECIFIED`, `URL_ENCODED`, `USER_PROVIDED`, so we'd probably want `TYPE_UNSPECIFIED` here as the default?

Less clear what the default should be for `custom_content_type` and `body` when not provided.

Reply:
Ah okay, [variables are `nullable: true` by default](https://developer.hashicorp.com/terraform/language/values/variables#disallowing-null-input-values), also:

> If you set an argument of a resource to null, Terraform behaves as though you had completely omitted it...

So that will work just fine, and I've confirmed this by updating my [PR](https://github.com/mozilla-it/webservices-infra/pull/2279) and `atlantis plan` says no changes.